src/DynamoCore/Models/NodeModel.cs

```diff
@@ -191,6 +191,54 @@ public bool IsUpstreamVisible
         }
 
         /// <summary>
+        /// Input nodes are used in Customizer and Presets. Input nodes can be numbers, number sliders,
+        /// strings, bool, code blocks and custom nodes, which don't specify path.
+        /// </summary>
+        public bool IsInputNode
```

Comment:
My intend here was next: if node doesn't have inports and it isn't imported node(i.e. ZeroTouch), then it's input node.
So, input nodes can be number sliders, strings, bool etc. And path nodes, e.g. directory path of file path.
I've done that, because path nodes can be uses in preset and can be specified as input node for preset.